pull_request_template.md

```diff
@@ -1,8 +1,3 @@
-## Read then delete
```

Comment:
is this no longer true?

Reply:
@evalsocket could you clarify - is this no longer true?

Reply:
ping

Reply:
it is coming from boilerplate, I didn't make any changes in pull_request template